pkgs/applications/office/espanso/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkgconfig
+, extra-cmake-modules
+, libX11
+, libXi
+, libXtst
+, libnotify
+, openssl
+, xclip
+, xdotool
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "espanso";
+  version = "0.5.5";
+
+  src = fetchFromGitHub {
+    owner = "federico-terzi";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0hdc83hzcngbzdng0ia7q1hcjss42dapanaibj6i93fcir84rnqp";
+  };
+
+  cargoSha256 = "1gmysgyacnkhhpq7lybkca6gdzsivj4dgppx0k7fb17pdqwzn9nk";
+
+  nativeBuildInputs = [
+    extra-cmake-modules
+    pkgconfig
+  ];
+
+  buildInputs = [
+    libX11
+    libXtst
+    libXi
+    libnotify
+    openssl
+    xdotool
+  ];
+
+  cargoBuildFlags = [ "--locked" ];
+
```

Comment:
Builds without this just fine for me. Is there a reason this is necessary? Otherwise:

```suggestion
```

Reply:
None that I am aware of. I just noticed the [aur package](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=espanso) had this flag.

I also managed to build and run without this flag.

Reply:
I thus removed the extra line.